tests/functional/ext/typing/redundant_union_assign_typehint_py38.py

```diff
@@ -0,0 +1,16 @@
+""""Checks for redundant Union typehints in assignments - python 3.8"""
```

Comment:
`from __future__ import annotations` was added in Python `3.7`.

Reply:
so what should be the difference between python 3.7 and 3.8 tests then? should I just merge them?

Reply:
> so what should be the difference between python 3.7 and 3.8 tests then? should I just merge them?

👍🏻 At the moment, the only difference is the future import. However, for the cases here it doesn't actually matter.

_You could even combine all tests into one file, if you keep the future import and add this:_
```ini
[TYPING]
runtime-typing=no
```

Reply:
I merged py37 and py38 it indeed worked. 

The `runtime-typing=no` option in the `rc` file works but with a little issue:
pylint with python 3.7 does not know 'deprecated-typing-alias', 'consider-alternative-union-syntax' and cannot parse them in the `# pylint: disable=` line.
pylint with python 3.10 knows all messages and was fine.

So I decided to keep 2 test versions for now. If you got any simple way to solve this (e.g. ignore certain message for certain python version) I could merge them into one, if it is important..

Reply:
> python 3.7 does not know 'deprecated-typing-alias', 'consider-alternative-union-syntax' and cannot parse them in the # pylint: disable= line.

It might be a bug but also it's possible that you need to set the py-version to 3.7 in the rc file.

Reply:
solved now